Libraries/libcar/Headers/car/Rendition.h

```diff
@@ -72,20 +72,49 @@ class Rendition {
         { return _format; }
     };
 
+public:
+    enum resize_mode {
```

Comment:
Agree in general. But I was thinking that would probably be a more C++ style API — this looked more like C so I wasn't sure which it was.